roles/user-management/manage-atlassian-users/README.md

```diff
@@ -16,7 +16,7 @@ atlassian:
   username: example
   password: example
 
-  user:
+  users:
```

Comment:
PR #295 changes these aspects of the `manage-atlassian-users` role, so if you could remove these changes from your PR it would make it easier on the merge.